x-pack/plugin/ml/qa/native-multi-node-tests/build.gradle

```diff
@@ -52,6 +52,10 @@ testClusters.all {
   setting 'xpack.ml.min_disk_space_off_heap', '200mb'
   setting 'indices.lifecycle.history_index_enabled', 'false'
   setting 'slm.history_index_enabled', 'false'
+  // To spice things up a bit, one of the nodes is not an ML node
+  nodes.'javaRestTest-0'.setting 'node.roles', '["master","data","ingest"]'
+  nodes.'javaRestTest-1'.setting 'node.roles', '["master","data","ingest","ml"]'
+  nodes.'javaRestTest-2'.setting 'node.roles', '["master","data","ingest","ml"]'
```

Comment:
We can't use an internal cluster test to test this functionality, because internal cluster tests cannot run native processes.  Instead, making one of the nodes in the multi-node cluster a non-ML node means one third of the `_explain` requests made by the test harness will go to it, and this ensures we have some test coverage for this change.